components/ToastMessage.js

```diff
@@ -36,10 +40,8 @@ const ToastWrapper = styled.div`
   z-index: 1000;
 `;
 
-const ToastMessage = ({ message, visible }) => {
+export default function ToastMessage({ message, visible }) {
   return visible ? (
     <ToastWrapper visible={visible}>{message}</ToastWrapper>
```

Comment:
this seems a bit redundant. if the prop visible is false, this is never rendered, therefore visible on the Wrapper will always be true. So you don't need the logic inside the ToastWrapper for visibility OR you don't need this function logic around it.

Reply:
Thanks! I have updated it here: https://github.com/Jennifer-Hinrichsen/emotion-tracker/pull/57/commits/1f479ee955b2f6691205c0b7d2498dc0fdc2a82a